key-events.go

```diff
@@ -81,7 +81,7 @@ func (p *keyeventPlugin) sendKeyEvent(window *glfw.Window, key glfw.Key, scancod
 		ScanCode:  scancode,
 		Modifiers: int(mods),
 	}
-	_, err := p.keyEventChannel.Send(event)
+	err := p.keyEventChannel.Send(event)
```

Comment:
Well, we are still v0.x (alpha) so breaking changes are allowed I guess.. Fix is easy and probably this isn't used often yet.